File: backend-portifolio/Models/SocialMediaAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend_portifolio.Models
{
    public class SocialMediaAccount
    {
        public long Id { get; set; }
        public string Provider { get; set; } 
        public string SocialId { get; set; }
        public string Email { get; set; }
        public string Username { get; set; }
        public SocialMediaAccount(string provider, string socialId, string email, string username)
        {
            Provider = provider;
            SocialId = socialId;
            Email = email;
            Username = username;
        }
    }
}

File: portfolio-v3/src/domain/models/SocialMediaAccount.ts
import { User } from "./User";

export class SocialMediaAccount{
    public Id: number;
    public Provider: string;
    public SocialId: string;
    public Email: string;
    public Username: string;
    public UserId : number;
    public User: User;
    constructor(){
        this.Id = 0;
        this.Provider = "";
        this.SocialId = "";
        this.Email = "";
        this.Username = "";
        this.UserId = 0;
        this.User = new User();
    }
}